internal/policy/commit/commit.go

```diff
@@ -57,6 +57,9 @@ type Commit struct {
 	DCO bool `mapstructure:"dco"`
 	// GPG enables the GPG signature check.
 	GPG bool `mapstructure:"gpg"`
+	// GPGSignatureGitHubOrganization enforces that GPG signature should come from
+	// one of the members of the GitHub org.
+	GPGSignatureGitHubOrganization string `mapstructure:"gpgSignatureGitHubOrg"`
```

Comment:
doesn't look short to me, but I don't have any better options? 

should it be subkey under `gpg`?

Reply:
I personally would like `gpg` do be more than just a `bool`. Will be a good place to put all GPG related things. But I know that can/will be a PITA. I could go either way.

Reply:
I guess we can't make it backwards compatible?

Reply:
No, but that hasn't stopped us in the past. There is a reason this project is still alpha 😄

Reply:
I don't see a way to change `gpg: true` to `gpg:\n  enabled: true` without breaking backwards compat or doing two structs and decoding into both of them...

Reply:
Yeah, what I mean it I am not against a breaking change. Conform is still alpha for that reason.

Reply:
got you, I will make a change then